### PR TITLE
VideoPress: add plan price on intro page

### PR DESCRIPTION
**Pull request by thedebian**
Related to 1668-gh-Automattic/greenhouse

## Proposed Changes

This PR pulls the premium or business plan pricing on the VideoPress onboarding intro page.

## Testing Instructions

* Apply pr
* `yarn start`
* Go to `/setup/videopress/intro`
* ✅ You should see a text "Starts at \<price\>" appearing
* ✅ The price should make sense in your locale

Capture d’écran 2023-04-12 à 10 40 39

### CODE REVIEW

**Comment by roundhill** [comment]
As discussed in Slack, let's try moving it below the buttons, all on one line using the normal body font and size.

File: client/landing/stepper/declarative-flow/internals/steps-repository/intro/index.tsx [hunk 1]
```diff
@@ -17,6 +24,44 @@ import './styles.scss';
...
+	const locale = useLocale();
+	const { supportedPlans } = useSupportedPlans( locale, 'ANNUALLY' );
+	const getPlanProduct = useSelect(
+		( select ) => ( select( PLANS_STORE ) as PlansSelect ).getPlanProduct,
+		[]
+	);
+	// VideoPress: we should always send a non-empty string so the spacing stays the same on the intro page
+	let videoPressGetStartedText: string | ReactElement = createInterpolateElement( '<nbsp />', {
+		nbsp: <>&nbsp;</>,
+	} );
```
Comment (roundhill) [inline client/landing/stepper/declarative-flow/internals/steps-repository/intro/index.tsx:36]: This is hilarious :D

**Review by roundhill — APPROVED** [review]
Looks good!

File: client/landing/stepper/declarative-flow/internals/steps-repository/intro/index.tsx (hunk 1 above)
Comment (thedebian) [reply client/landing/stepper/declarative-flow/internals/steps-repository/intro/index.tsx:36]: Yeah that's quite a bunch of code for a whitespace 😅

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/7922118

Thank you @thedebian for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/7922118).

**Pull request merged**